doc/script_commands.txt

```diff
@@ -8037,7 +8037,7 @@ false means OFF, and true means ON. See setmapflag() for examples of mapflags.
 *getbattleflag("<battle flag>")
 
 Sets or gets the value of the given battle flag.
-Battle flags are the flags found in the conf/map/battle/*.conf files and is
+Battle flags are the flags found in the conf/map/battle/\*.conf files and is
```

Comment:
```suggestion
Battle flags are the flags found in the conf/map/battle/*.conf files and is
```

Reply:
I change this because `/*` make entire code as comment.

Reply:
I would say that's the fault of the code editor, as it's a txt file it should not auto-detect C comments.
This change may cause some legibility problems, as it's not clear what's the purpose of that backslash in a path.